src/lib/y2storage/boot_requirements_strategies/prep.rb

```diff
@@ -55,6 +62,36 @@ def warnings
 
     protected
 
+      # PReP partition is needed, so return any warning related to it.
+      def prep_warnings
+        res = []
+        big_preps = too_big_preps
+
+        if missing_partition_for?(prep_volume)
+          res << SetupError.new(missing_volume: prep_volume)
+        elsif !big_preps.empty?
+          res << SetupError.new(message: big_prep_warning(big_preps))
+        end
+
+        res
+      end
+
+      def big_prep_warning(big_partitions)
+        # TRANSLATORS: %s is single or list of partitions that are too big.
+        msg =
+          format(
+            n_(
+              "Following PReP partition is too big: %s. ",
+              "Following PReP partitions are too big: %s.",
```

Comment:
This can stand as it is because omitting the initial "the" in this kind of texts is becoming kind of usual in computer messages (that's why I didn't notice at first sight, or maybe I have been too exposed to you :wink:), but for it to be perfect English it should be "The following..."